pkg/env/prod.go

```diff
@@ -43,6 +43,7 @@ type prod struct {
 	domain                   string
 	serviceKeyvaultURI       string
 	vnetName                 string
+	peVnetName               string
```

Comment:
rather than do this, let's go in the opposite direction of removing the tags on the vnets + hard-coding the vnet names - so remove `vnetName`.

Reply:
removed vnetName and peVnetName.